dashboard/app/models/pd/teacher_application.rb

```diff
@@ -95,9 +95,15 @@ def validate_required_application_fields
     end
   end
 
+  before_validation :downcase_emails
+  def downcase_emails
```

Comment:
Though I did just realize I don't need to downcase on line 410 since it's already done here. Will update.